### PR TITLE
Added external styling for propdrawer & playground

### PR DESCRIPTION
**Pull request by shivambalwani**
issue #3 :
Added External Styling for the Components Propdrawer and Playground.

### CODE REVIEW

File: src/components/PropDrawer/index.jsx [hunk 1]
```diff
@@ -41,13 +37,7 @@ export default function TemporaryDrawer(props) {
...
           onClose={toggleDrawer("right", false)}
         >
-          <div
-            style={{
-              width: "70vw",
-              maxWidth: 700,
-              paddingLeft: "1rem",
-            }}
-          >
+          <div className={styles.propdrawer__child}>
```
Comment (tewarig) [inline src/components/PropDrawer/index.jsx:40]: just a suggestion we can use camel casing here 😅 ... I remembered seeing a pr in which the maintainer asked to use camel casing.

Comment (tewarig) [inline src/components/PropDrawer/index.jsx:40]: just a suggestion we can use camel casing here  ... I remembered seeing a pr in which the maintainer asked to use camel casing.

**Review by Uyadav207 — COMMENTED** [review]
Screenshot from 2021-03-13 22-42-01


- Breaking UI.

**Comment by Uyadav207** [comment]
I think if you are changing UI, please make a discussion for that particular UI issue. This will definitely save unwanted maintainers from a hard time maintaining the repo.

***Possible Solution***

- prepare `mockups`.
- discuss with maintainer @pranshuchittora.
- Then Make PR

I hope it makes any sense. 
Thanks.

**Comment by tewarig** [comment]
> I think if you are changing UI, please make a discussion for that particular UI issue.
> _**Possible Solution**_
> 
> * prepare `mockups`.
> * discuss with maintainer @pranshuchittora.
> * Then Make PR
> 
> I hope it makes any sense.
> Thanks.

I tried to see what is causing this UI to break. 
have a look at the elements.style in this pr
Screenshot from 2021-03-13 23-04-33
elements.style in the main deployment. 
Screenshot from 2021-03-13 23-00-09
just figure out a way to add these styles. hope it helps.

**Comment by shivambalwani** [comment]
> > I think if you are changing UI, please make a discussion for that particular UI issue.
> > _**Possible Solution**_
> > 
> > * prepare `mockups`.
> > * discuss with maintainer @pranshuchittora.
> > * Then Make PR
> > 
> > I hope it makes any sense.
> > Thanks.
> 
> @shivambalwani
> I tried to see what is causing this UI to break.
> have a look at the elements.style in this pr
> Screenshot from 2021-03-13 23-04-33
> elements.style in the main deployment.
> Screenshot from 2021-03-13 23-00-09
> just figure out a way to add these styles. hope it helps.

thanks!

File: src/components/playground/playground.module.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.playground{
```
Comment (pranshuchittora) [inline src/components/playground/playground.module.css:1]: can you rename it to something container

File: src/components/PropDrawer/index.jsx (hunk 1 above)
Comment (pranshuchittora) [reply src/components/PropDrawer/index.jsx:40]: And pls remove the `child` and rename to something more intuitive

**Review by pranshuchittora — CHANGES_REQUESTED** [review]
Left some comments

**Comment by shivambalwani** [comment]
> Left some comments

@pranshuchittora i have fixed the style naming, but the checks are failing. I don't know why?

**Comment by pranshuchittora** [comment]
Oops!
Pls pull the new changes & resolve the conflicts.

**Comment by shivambalwani** [comment]
> Oops!
> Pls pull the new changes & resolve the conflicts.

pulled the new changes still unable to resolve

**Pull request closed without merging**